Git-commit: 76d413e524a9a48ad96a33f18d54029f9bd610cc
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 26 Jun 2020 15:53:44 +0100
Subject: [PATCH] media: bcm2835-unicam: Avoid gcc warning over {0} on endpoint
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

raspberrypi inclusion
category: feature
bugzilla: 50432

--------------------------------

Older gcc versions object to = { 0 } initialisation if the first
elemtn in the structure is a substructure.

Use = { } to avoid this compiler warning.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
Signed-off-by: Fang Yafen <yafen@iscas.ac.cn>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 drivers/media/platform/bcm2835/bcm2835-unicam.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/bcm2835/bcm2835-unicam.c b/drivers/media/platform/bcm2835/bcm2835-unicam.c
index 9adfb2d217af..a65e1da55142 100644
--- a/drivers/media/platform/bcm2835/bcm2835-unicam.c
+++ b/drivers/media/platform/bcm2835/bcm2835-unicam.c
@@ -2586,7 +2586,7 @@ static const struct v4l2_async_notifier_operations unicam_async_ops = {
 static int of_unicam_connect_subdevs(struct unicam_device *dev)
 {
 	struct platform_device *pdev = dev->pdev;
-	struct v4l2_fwnode_endpoint ep = { 0 };
+	struct v4l2_fwnode_endpoint ep = { };
 	struct device_node *ep_node;
 	struct device_node *sensor_node;
 	unsigned int lane;
-- 
2.31.1

